src/contrib/mlir/pass/mlir_subgraph_extraction.cpp

```diff
@@ -291,6 +334,10 @@ bool MLIRSubgraphExtractionPass::is_supported_mlir_op(std::shared_ptr<Node> node
         {
             return false;
         }
+        else
+        {
+            return true;
+        }
```

Comment:
super-nit: we wouldn't need `else` after `if` with `return`. Not sure if this is in nGraph guidelines or it's just an LLVM thing.